tools/prove_wrapper

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+OUTPUT=$(mktemp)
+
+echo "Running prove with TAP output check ..."
+
+prove "$@" 2>&1 | tee "$OUTPUT"
+STATUS=${PIPESTATUS[0]}
+
+if [ "$STATUS" -ne 0 ]; then
+    echo "not ok - prove failed"
+    exit "$STATUS"
+fi
+
+sed -E '
+  s/^\[[0-9]{2}:[0-9]{2}:[0-9]{2}\][[:space:]]*//
+  /^All tests successful\./,$d
+' "$OUTPUT" > "$OUTPUT.processed"
+
+UNHANDLED=$(grep -vE '^t/.*([0-9]{2}-|)([[:alpha:]]|_|-)+\.t \.+' "$OUTPUT.processed")
+
+if [ -n "$UNHANDLED" ]; then
+    echo "not ok - unhandled output found"
```

Comment:
```suggestion
    echo "not ok - unhandled output found"
    echo "Run with PROVE_COMMAND='tools/prove_wrapper' to reproduce locally" "
```

Reply:
It seems like you added this to the successful case now?

Reply:
Moved into the unsuccessful case :grin: